components/LoginForm/LoginForm.js

```diff
@@ -64,16 +69,18 @@ function LoginForm({ initialValues, login, onSuccess }) {
             <Field
               type="email"
               name="email"
-              label="Email*"
+              label={redirectFunc ? '' : 'Email*'}
```

Comment:
Hmm, we can't do this, because it's an accessibility violation. Same goes for the password field.

I think the wireframe/sketch didnt have labels, but it's totally fine that we have them.

Reply:
✔️